### DISCUSSION TITLE
Unverified Developer for latest version?

### DISCUSSION THREAD

**Discussion by OogieM** [pawelsalawa/sqlitestudio#5133] [Q&A] [answered]
I went to upgrade the app and get this error message. "“SQLiteStudio-3.4.6-osx-installer.app” cannot be opened because the developer cannot be verified." 

The default on Macs is Mac App store only but I already have the app store and verified developers options set. 

I thought you were an identified developer. Any plans to become one?

**Comment by pawelsalawa** [comment, marked as answer]
As mentioned in #3932

> After some more research, obtaining certificates, renewing and even covering it from technical perspective - takes lots and lots of effort. Not to mention money (not so little). Here's even a post about this: https://virtualgl.org/DeveloperInfo/CodeSigningHell
> 
> It appears that what MS and Apple are doing is pretty similar to what ATARI tried to do at the time (1970's), i.e. to prevent anyone else from releasing software for their platform without their (paid) blessing.
> 
> For now I'm parking this ticket with hope, that they (MS and Apple) will come to common sense and make Open Source signing less painful and more affortable (the latter one especially related to Apple). I hope for "Let's Encrypt" for code signing. There already is initiative https://sigstore.dev/ , but at the moment it's for Linux only. We will see what future brings.

From time to time I check on this whether the situation improved, but at the moment it does not seem to.

**Reply by OogieM** [reply]
OK, Thanks
